documentation/website/docs/installation.md

```diff
@@ -54,6 +54,22 @@ This will generate a `Makefile` in your checkout directory. You can subsequently
 $ make
 $ make test
 ```
+### Testing changes to the Python Client
+In a virtualenv, install dependencies with `requirements.txt` and run python tests to make sure everything is set up correctly
+
+```bash
+$ cd /path/to/pyre-check
+$ pip install -r requirements.txt
+$ ./scripts/run-python-tests.sh
+```
+When installing and running `pyre` from PyPi, the entry point to the executable is actually `client/pyre.py`. To be able to run this file from anywhere, add the `pyre-check` directory to `PYTHONPATH` and subsequently assign `client.pyre` an alias to invoke the binary from anywhere
+
+```bash
+$ echo "export PYTHONPATH=/path/to/pyre-check" >> ~/.bashrc
+$ echo "alias pyre='python -m client.pyre'" >> ~/.bashrc
```

Comment:
Just a suggestion: to avoid polluting the python path of the user, we could use:
```
echo "alias pyre='PYTHONPATH=/path/to/pyre-check python -m client.pyre'" >> ~/.bashrc
```

Reply:
Ohhh I like this even better! Good suggestion.

On top of that change, we probably don't want to overwrite the exiting pythonpath, so maybe augment it using `:$PYTHONPATH`
```suggestion
$ echo "alias pyre='PYTHONPATH=/path/to/pyre-check python:$PYTHONPATH -m client.pyre'" >> ~/.bashrc
```

Reply:
Good point. I think it should be:
```
$ echo "alias pyre='PYTHONPATH=\"/path/to/pyre-check:$PYTHONPATH\" python -m client.pyre'" >> ~/.bashrc
```
I think the quote is required if there is any space in the path.